docs/docs/development/new-shield.md

```diff
@@ -115,10 +115,18 @@ endif
 
 ## Shield Overlays
 
+### Pro Micro Compatible Shields
```

Comment:
It might be nice to make these two tabs that can be switched between; you can see an example right below for unibody/split shields.